packages/server/app.js

```diff
@@ -4,14 +4,11 @@ const path = require('path');
 const fs = require('fs');
 const cookieParser = require('cookie-parser');
 const logger = require('morgan');
-const firebase = require('firebase-admin');
 
-const app = express();
+// Firebase boilerplate.
+const firebase = require('firebase-admin');
 
-// disable when running unit tests
-if (process.env.NODE_ENV !== 'test') {
-  app.use(logger('dev'));
-}
+const serviceAccount = require('./service_account.json');
```

Comment:
If we import firebase credentials using require, we don't need line 16. The require call will fail if the file does not exist.